feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewScreen.kt

```diff
@@ -123,18 +132,21 @@ fun PreviewScreen(
                         modifier = Modifier
                             .align(Alignment.TopCenter),
                         isOpen = previewUiState.quickSettingsIsOpen,
-                        toggleIsOpen = { viewModel.toggleQuickSettings() },
+                        toggleIsOpen = {
```

Comment:
Looks like there is a lot of unrelated reformatting here and in other files making it harder to review the actual changes for the tests. Please revert unrelated changes for now.

Reply:
These reformatting are necessary for spotless check

Reply:
Ok. Looks like we'll be running into these for a while until we've touched all the files. Unfortunate but it should settle eventually.